### DISCUSSION TITLE
IOS

### DISCUSSION THREAD

**Discussion by TITANIUM339** [lnreader/lnreader#1047] [Q&A] [answered]
Has anyone considered making an IOS version?

**Comment by rajarsheechatterjee** [comment, marked as answer]
Being a React Native app, it should have supported both platforms ideally. However, due to the lack of an iOS device or Mac during development, the iOS files were removed. To add iOS support now, we may need to restart the project from scratch, which would require significant effort.

**Comment by nyagami** [comment]
Not only development, but also distribution. The official way is publishing LNReader on App Store which require an Apple Developement account (99$/year) and strict reviewed by App Store (could not pass with this kind of app). Or Users have to use alternative ways to install the .ipa file (jailbreak, sideloading = refresh/reinstall app frequently).

**Comment by TITANIUM339** [comment]
Although it is true that Apple makes distribution difficult as @nyagami mentioned, I think this may soon change due to EU regulation. Anyway, Thank you guys for taking the time to answer my question.
